roles/ceph-mds/tasks/create_mds_filesystems.yml

```diff
@@ -1,10 +1,19 @@
 ---
-- name: create filesystem pools
-  command: "{{ hostvars[groups[mon_group_name][0]]['docker_exec_cmd'] | default('') }} ceph --cluster {{ cluster }} osd pool create {{ item.name }} {{ item.pgs }}"
-  changed_when: false
-  delegate_to: "{{ groups[mon_group_name][0] }}"
-  with_items:
-    - "{{ cephfs_pools }}"
+- name: filesystem pools related tasks
+  block:
```

Comment:
Wrong indent in this `block`?

Reply:
Indent was OK. It was an issue with the kind of quote I had used.